src/utils/paginationHandler.js

```diff
@@ -1,130 +1,139 @@
-        pageIdentifier = paginator.maxNumberOfPages - 1;
-    }
-
-    if (pageIdentifier < 0) {
-      pageIdentifier =
-        paginator.maxNumberOfPages +
-        (pageIdentifier % paginator.maxNumberOfPages);
-    } else if (pageIdentifier >= paginator.maxNumberOfPages) {
-      pageIdentifier %= paginator.maxNumberOfPages;
-    }
-    return { ...paginator.currentIdentifiers, pageIdentifier };
-  };
-
-  const buttonPaginator = new ButtonPaginator(interaction, {
-    pages,
-    buttons,
-    identifiersResolver
-  });
-  buttonPaginator.on(PaginatorEvents.PAGINATION_READY, (paginator) => {
-    for (const actionRow of paginator.messageActionRows) {
-      for (const button of actionRow.components) {
-        button.disabled = false;
-      }
-    }
-    return interaction.editReply(paginator.currentPage);
-  });
-  buttonPaginator.on(PaginatorEvents.COLLECT_ERROR, async () => {
-    try {
-      /**
-       * @type {Message}
-       */
-      const message = await interaction.fetchReply();
-
-      /**
-       * @type {MessageEmbed}
-       */
-      const embed = message.embeds[0];
-
-      const components = message.components;
-      components[0].components.forEach((component) => {
-        component.disabled = true;
-      });
-      embed.setColor(config.COLORS.ERROR);
-      return interaction.editReply({
-        embeds: [embed],
-        components: components
-      });
-    } catch (error) {
-      if (error.code) return;
-      return console.log(error);
-    }
-  });
-  buttonPaginator.on(PaginatorEvents.PAGINATION_END, async () => {
-    try {
-      /**
-       * @type {Message}
-       */
-      const message = await interaction.fetchReply();
-
-      /**
-       * @type {MessageEmbed}
-       */
-      const embed = message.embeds[0];
-      const components = message.components;
-      components[0].components.forEach((component) => {
-        component.disabled = true;
-      });
-      embed.setColor(config.COLORS.ERROR);
-      return interaction.editReply({
-        embeds: [embed],
-        components: components
-      });
-    } catch (error) {
-      if (error.code) return;
-      return console.log(error);
-    }
-  });
-
-  await buttonPaginator.send();
-  return buttonPaginator.message;
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
 };
+<<<<<<< Updated upstream
```

Comment:
Merge conflict here as well

Reply:
@Juilzze resolved this issue in latest commit.